pkgs/servers/radarr/default.nix

```diff
@@ -17,6 +17,7 @@ stdenv.mkDerivation rec {
 
     makeWrapper "${mono}/bin/mono" $out/bin/Radarr \
       --add-flags "$out/share/${name}/Radarr.exe" \
+      --prefix LD_LIBRARY_PATH ':' "${curl.out}/lib" \
```

Comment:
Not a blocker, but you could take the opportunity to combine these into a single `makeLibraryPath [ curl sqlite ... ]` here. Briefer and obviates the need to worry about output selection.